quiz/src/main/java/com/epam/spring/homework3/quiz/controller/GameController.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.spring.homework3.quiz.controller;
+
+import com.epam.spring.homework3.quiz.controller.dto.QuizDto;
+import com.epam.spring.homework3.quiz.service.GameService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+@RequestMapping("/game")
+@RequiredArgsConstructor
+@Api(tags = "GameController description for SWAGGER documentation")
+public class GameController {
```

Comment:
This one is also interesting. I would suggest next url format.

Also remember, GET requests should not modify or write the new data, it's just used to retrieve the data


`POST /game/{id}/start`

`GET /game/{id}/user/{userId}/result`